docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -133,40 +149,49 @@ following commands from `spark-shell`, assuming Alluxio Master is running on `lo
 ```scala
 > val s = sc.textFile("alluxio://localhost:19998/LICENSE_HDFS")
 > val double = s.map(line => line + line)
-> double.saveAsTextFile("alluxio://localhost:19998/LICENSE_HDFS2")
+> double.saveAsTextFile("alluxio://localhost:19998/Output_HDFS2")
 ```
 
 Open your browser and check [http://localhost:19999/browse](http://localhost:19999/browse). There
 should be an output `LICENSE_HDFS2` which doubles each line in the file `LICENSE_HDFS`. Also, the
 `LICENSE_HDFS` file now appears in the Alluxio file system space.
```

Comment:
`LICENSE_HDFS2` -> `Output_HDFS2` and similar wording about the directory

Reply:
fixed